frontend/pkg/database/database.go

```diff
@@ -94,7 +100,7 @@ func (d *CosmosDBClient) DBConnectionTest(ctx context.Context) error {
 }
 
 // GetClusterDoc retrieves a cluster document from async DB using resource ID
-func (d *CosmosDBClient) GetClusterDoc(ctx context.Context, resourceID string, partitionKey string) (*HCPOpenShiftClusterDocument, error) {
+func (d *CosmosDBClient) GetClusterDoc(ctx context.Context, resourceID string, subscriptionID string) (*HCPOpenShiftClusterDocument, error) {
```

Comment:
+1 - This is clearer.  Yes we're using subscription ID as the partition key but that should remain an internal implementation detail.